docker-compose.katana.yaml

```diff
@@ -21,3 +21,5 @@ services:
     environment:
       - ACCOUNT_ADDRESS=0x03ee9e18edc71a6df30ac3aca2e0b02a198fbce19b7480a63a0d71cbd76652e0
       - PRIVATE_KEY=0x0300001800000000300000180000000000030000000000003006001800006600
+      - CHECK_INTERVAL=0.1
+      - MAX_WAIT=5
```

Comment:
Just adding this here: https://github.com/kkrt-labs/kakarot/pull/659/files#diff-af3d9b2bc82592f9097e7fda59b3f253128e787f0387f81508cb0630c2c48bb6R93-R94